### ISSUE TITLE
{summary} order should be the same as visual order of fields

### ISSUE THREAD

**Issue by kevinwhoffman** [CalderaWP/Caldera-Forms#241]
My form fields are:
- first_name
- last_name
- email
- phone
- comments_questions

...but the {summary} in the email reports comments_questions before phone. I don't know if this is relevant but I used the contact form starter template and then added the phone field into it, so phone was the only field that wasn't already part of the template.

**Comment by Shelob9** [comment]
@kevinwhoffman Yes, it's going to do it in the order the fields were added, but it should be the way you expect it. Not sure how tricky this would be to do. @Desertsnowman would know better.

In the mean time, you should probably skip the `{summary}` tag and use the individual magic tags instead:

```
%first_name%
%last_name%
%email%
%phone%
%comments_questions%
```

**Comment by kevinwhoffman** [comment]
@Shelob9 Thanks for the explanation. Manually setting the tags is working fine.

**Comment by Shelob9** [comment]
This was fixed during 1.4.x cycle
